src/index.html

```diff
@@ -4,9 +4,383 @@
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com/"
+    crossorigin="true"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com/" >
+    <link rel="stylesheet" href="styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          class="header__logo" 
+          alt="images_logo" 
+          src="images/logo.png"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-el">
+            <a class="nav__list-link is-active" href="#Apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Samsung" class="nav__list-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Smartphones" class="nav__list-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-el" data-qa="nav-hover">
+            <a href="#Laptop&Computers" class="nav__list-link">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Gadget" class="nav__list-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Tablets" class="nav__list-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Photo" class="nav__list-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Video" class="nav__list-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <div data-qa="card" class="card">
+        <img 
+          class="card__picture" 
+          src="images/imac.jpeg" 
+          alt="iMacPic"
+        >
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <p class="stars__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__cost">
+          <p class="card__price">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a 
+          data-qa="card-hover" 
+          class="card__buy" 
+          href="#buy"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </div>
      
      <div class="card">
```

Reply:
Not fixed 😢 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/1eb335cf-9caa-408a-a703-b7d8400efa54)